### DISCUSSION TITLE
Using Lua?

### DISCUSSION THREAD

**Discussion by ghost** [maritz/nohm#165] [Ideas]
Why not use lua scripts to do all the indexing an saving? Wouldn't that be faster?

**Comment by maritz** [comment]
Sorry I haven't answered yet. I'm still thinking that I'll do a 1.0.0 some day in the future and this was one of the points I definitely wanted to look into.

If you don't mind, I'll reopen this and then let you know once I get around to it. Or alternatively if you have the time, I'd be happy about some analysis reports and/or pull requests. :-)
